internal/reconciliations/gateway/kyma_gateway.go

```diff
@@ -95,6 +95,10 @@ func reconcileGardenerKymaGateway(ctx context.Context, k8sClient client.Client,
 		return err
 	}
 
+	if err := reconcileKymaGatewayVirtualService(ctx, k8sClient, apiGatewayCR, domain); err != nil {
```

Comment:
Right now this is also applied for non-Gardener clusters, we should keep the same behaviour. Also I think there is no harm in doing it the same way, even if it might not be used for open source.

Reply:
Added to non gardener reconcile function.